src/content-tags-drawer/ContentTagsCollapsible.scss

```diff
@@ -5,6 +5,10 @@
   .collapsible-trigger {
     border: none !important;
   }
+
+  .tags-tree {
+    font-size: 17px;
```

Comment:
I think it's generally a bad idea to use px sizes since they don't zoom well, perhaps we can use the closest css class or rem value?

Reply:
Updated here: https://github.com/openedx/frontend-app-course-authoring/pull/939/commits/a185fe7cb6ad4bb5fb94698135a8d903af3776f6